### PR TITLE
Add CI builds (MacOS only) for ANGLE

### PR DESCRIPTION
**Pull request by ramezgerges**
# Summary of the PR
This PR adjusts the `Angle.cs` script to optimize disk space usage and also build fat binaries for MacOS (to work on both arm64 and x64 macs). For now, only MacOS builds are enabled.

# Related issues, Discord discussions, or proposals
Links go here.
https://github.com/dotnet/Silk.NET/issues/2024
# Further Comments

### CODE REVIEW

**Comment by ramezgerges** [comment]
@dotnet-policy-service agree [company="Uno Platform Inc."]

**Comment by ramezgerges** [comment]
@dotnet-policy-service agree company="Uno Platform Inc."

**Comment by jeromelaban** [comment]
Hello @Perksey! We're looking to making those modifications for Angle arm64, you can find us on Discord to further discuss it as needed, thanks!

File: .github/workflows/angle.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: MoltenVK
```
Comment (Perksey) [inline .github/workflows/angle.yml:1]:
```suggestion
name: ANGLE
```

**Review by Perksey — APPROVED** [review]
Excellent, thank you! I have verified this is working as intended: #2291

**Pull request merged**